e2e/tests/addHearingBookingDetails_test.js

```diff
@@ -0,0 +1,87 @@
+const config = require('../config.js');
+const hearingDetails = require('../fixtures/hearingDetails.js');
+
+let caseId;
+
+Feature('Add hearing booking details');
+
+Before(async (I, caseViewPage, submitApplicationEventPage) => {
+  I.logInAndCreateCase(config.swanseaLocalAuthorityEmailUserOne, config.localAuthorityPassword);
+  caseId = await I.grabTextFrom('.heading-h1');
+  caseViewPage.goToNewActions(config.applicationActions.submitCase);
+  submitApplicationEventPage.giveConsent();
+  I.continueAndSubmit();
+  I.signOut();
+});
+
+Scenario('Enter hearing details and submitting as HMCTS admin', async (I, caseViewPage, loginPage, addHearingBookingDetailsEventPage) => {
+  loginPage.signIn(config.hmctsAdminEmail, config.hmctsAdminPassword);
+  I.navigateToCaseDetails(caseId);
+  I.see(caseId);
+  caseViewPage.goToNewActions(config.administrationActions.addHearingDetails);
+  addHearingBookingDetailsEventPage.addHearing();
+  await addHearingBookingDetailsEventPage.enterHearingDetails(hearingDetails[0]);
+  addHearingBookingDetailsEventPage.addHearing();
+  await addHearingBookingDetailsEventPage.enterHearingDetails(hearingDetails[1]);
+  I.continueAndProvideSummary('summary', 'description');
+  I.seeEventSubmissionConfirmation(config.administrationActions.addHearingDetails);
+  caseViewPage.selectTab(caseViewPage.tabs.hearings);
+  seeAnswersInHearingTab(I);
```

Comment:
Why can't we use the `seeAnswerInTab` method in the steps_file?

Reply:
because i guess its specific to this hearing event ? is that ok to add it there? @Jamesnelson117

Reply:
Hmmm, I don't get why the seeAnswerInTab in steps file won't do the same thing?

Reply:
Happy with this Adam